jazz-ui/src/app/pages/service-logs/service-logs.component.ts

```diff
@@ -237,32 +166,31 @@ export class ServiceLogsComponent implements OnInit {
 	limitValue : number = 20;
 	offsetValue:number = 0;
 
-	// environmentList = ['dev', 'stg', 'prod'];
 	envList = ['prod','stg'];
 	
-	accList=['tmodevops','tmonpe'];
-  regList=['us-west-2', 'us-east-1'];
-	accSelected:string = 'tmodevops';
-  regSelected:string = 'us-west-2';
+	accList=env_internal.urls.accounts;
+	regList=env_internal.urls.accounts;
```

Comment:
same as above, look at every place where you have `regList`

Reply:
I have fixed this issue, Updated the PR